src/CMakeLists.txt

```diff
@@ -86,9 +86,11 @@ set(keepassx_SOURCES
     gui/FileDialog.cpp
     gui/IconModels.cpp
     gui/KeePass1OpenWidget.cpp
+    gui/kmessagewidget.cpp
```

Comment:
I think it should, but I left it as is, because that was the original file name. I guess I'll change it.